website/src/css/custom.css

```
@@ -1,632 +1,1225 @@
-/* ==========================================================================
-   Design System — @horizon-republic/nestjs-jetstream docs
-   Dark-only theme, built on Infima CSS custom properties.
-   ========================================================================== */
 
-/* ===== Design Tokens ===== */
+@import url("https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap");
```

Stylelint (17.9.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap"" (import-notation)

(import-notation)